backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Helpers/PolicyHelper.cs

```diff
@@ -157,6 +158,11 @@ public static string GetAltinnAppDelegationPolicyPath(string org, string app, st
                 {
                     throw new ArgumentException("App was not defined");
                 }
+
+                if (string.IsNullOrWhiteSpace(org) && string.IsNullOrWhiteSpace(app) && string.IsNullOrWhiteSpace(resourceRegistryId))
+                {
+                    throw new ArgumentException("ResourceRegistryId was not defined");
```

Comment:
Because of the order of these checks this will never happen as it would already have thrown exception for missing org

Reply:
This is necessary for the code to throw error of missing resourceRegistryId only when resourceRegistryId is empty. Without this check it threw the error also when org and app was set.

Reply:
Yes, but now this error will never be thrown because it requires org to be IsNullOrWhiteSpace to be true.
And if so, it will already have thrown the org missing argument.

I'm wondering whether the code should be structured differently for better separation of the org/app vs resourceId flows